app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -25,15 +26,16 @@ private const val FETCH_UNIQUE_WORK_NAME = "PocketFetch"
  * Schedules background fetches of the Pocket video data.
  */
 class PocketVideoFetchScheduler(
-    private val isPocketEnabledByLocale: () -> Boolean
+    private val isPocketEnabledByLocale: () -> Boolean,
+    private val context: Context
```

Comment:
nit: -> `context` -> `appContext`

Reply:
This will be removed since we are deprecating pocket stuff anyway